Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -1,10 +1,6 @@
 ---
 title: Prerequisites
-weight: 1000
 ---
-{% include_relative branch.liquid %}
-
-# Prerequisites
 
 Rook can be installed on any existing Kubernetes cluster as long as it meets the minimum version
```

Comment:
While you're in here can you change the min version to 1.17? It was missed in a previous change, thanks.